corehq/apps/data_interfaces/views.py

```diff
@@ -155,6 +155,69 @@ def get_deleted_item_data(self, item_id):
         }
 
 
+class ArchiveFormView(DataInterfaceSection):
```

Comment:
decided to throw the feature behind a feature flag. i think it'll make it easier to release later rather than mess with permissions